scripts/pylint_extensions_test.py

```diff
@@ -230,3 +230,32 @@ def test_finds_backslash_continuation(self):
             ),
         ):
             temp_file.close()
+
+
+class FunctionArgsOrderCheckerTest(unittest.TestCase):
+
```

Comment:
Hi @vibhor98, do we need docstrings here too, that is, for this test files? Thanks!

Reply:
Hi @apb7, yes we need to add docstrings to the methods in test files but only selected ones. Those ending with `*Test` (like this class name) or `test_*` don't need docstrings. We're handling this via regex.